r2/r2/public/static/css/highlight.css

```diff
@@ -8,7 +8,7 @@
 }
 
 /* github.com style (c) Vasily Polovnyov <vast@whiteants.net> - from the highlight.js source */
-pre {
+pre.subreddit-stylesheet-source {
```

Comment:
I think we should avoid editing this file since it's 3rd party stuff.  It really should be moved to `css/lib` or something like that.  I know it s a little annoying, but can you do the inverse (i.e add styles to _undo_ this file's effect) in `reddit.less`?